### ISSUE TITLE
screenshot

### ISSUE THREAD

**Issue by hharch** [freeplane/freeplane#1714]
want a screenshot function so that can add a screenshot from a web page to a mind map

**Comment by euu2021** [comment]
Do you mean, after the screenshot is taken, it is automatically added to the map?

**Comment by hharch** [comment]
> Do you mean, after the screenshot is taken, it is automatically added to the map?

Yes,every time I take a screenshot, I have to open other software, which is inconvenient
